instrumentation/docker/src/main/scala/net/cakesolutions/testkit/logging/docker/formats/LogEventFormat.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 Cake Solutions Limited
+// Copyright 2017 Carl Pulley
+
+package net.cakesolutions.testkit.logging.docker.formats
+
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+import scala.util.control.NonFatal
+
+import cats.syntax.either._
+import io.circe._
+import net.cakesolutions.testkit.logging.LogEvent
+
+/**
+  * Utility methods for transforming between log lines and JSON log events.
+  */
+object LogEventFormat {
```

Comment:
Is this Docker specific. If it's not, should it be in the `core` package? If it is, perhaps it should have a name that makes that clear?